README.md

```diff
@@ -22,18 +22,41 @@ forwardproxy {
     response_timeout 30
     dial_timeout     30
     upstream         https://user:password@extra-upstream-hop.com
+    acl {
+      allow     caddyserver.com$
+      deny      192.168.1.1/32 192.168.0.0/16 prohibitedsite.com$ ^localhost$
+      allow     ::1/128 8.8.8.8 ^allowedsite.com$
+      allowfile /path/to/whitelist.txt
+      denyfile  /path/to/blacklist.txt
+      allow     all
+      deny      all // unreachable rule, remaining requests are matched by `allow all`
+    }
 }
 ```
 
 (The square brackets `[ ]` indicate values you should replace; do not actually include the brackets.)
 
+##### Security
+
 - **basicauth [user] [password]**  
 Sets basic HTTP auth credentials. This property may be repeated multiple times. Note that this is different from Caddy's built-in `basicauth` directive. BE SURE TO CHECK THE NAME OF THE SITE THAT IS REQUESTING CREDENTIALS BEFORE YOU ENTER THEM.  
 _Default: no authentication required._
 
-- **ports [integer] [integer]...**  
-Whitelists ports forwardproxy will HTTP CONNECT to.  
-_Default: no restrictions._
+- **probe_resistance [secretlink.tld]**  
```

Comment:
rephrased this section a bit, other previously existing sections were not changed, except for the upstream, which had "Upstream is incompatible with `acl` and `ports` subdirectives." added to it.

Reply:
:+1: